### DISCUSSION TITLE
Is there any way to disable the tab key from selecting what my mouse had highlighted?

### DISCUSSION THREAD

**Discussion by p1pah** [adobe/react-spectrum#7045] [Q&A]
In all of these Combobox examples (https://react-spectrum.adobe.com/react-aria/ComboBox.html#content), if I hover over one of the combobox options with my mouse and then tab out, it selects what was highlighted. Is there any way to make this arrow key only behavior? I looked in the RAC source code and it looks like it just selects whatever has the focus. I think it is working correctly but just curious if I can disable the functionality of focusing with my mouse and making it arrow key specific for the focus...? 
Also, any reason this behaves differently to https://react-spectrum.adobe.com/react-aria/Select.html#content...?

**Comment by p1pah** [comment]
Is the shouldFocusOnHover here the potential culprit?
![image](https://github.com/user-attachments/assets/191b5b67-a683-44cb-897b-2304c9ec2968)

**Comment by romansndlr** [comment]
If you find out please let me know 🙏

**Reply by p1pah** [reply]
@romansndlr This PR would fix it....https://github.com/adobe/react-spectrum/pull/7163
